src/services/message.service.ts

```diff
@@ -77,25 +77,30 @@ export class MessageService {
     return msg;
   }
 
-  public generateEmbedList(title: string, listItems: string[]): MessageEmbed {
+  public generateEmbedList(listItems: string[], arg: MessageEmbed | string): MessageEmbed {
     const maxCol = 3;
     const maxRows = 10; // This is a soft limit
 
     // Setting up number of columns and rows for the list
     const numCols = Math.min(maxCol, Math.ceil(listItems.length / maxRows));
     const numRows = Math.ceil(listItems.length / numCols);
 
-    // Sets up the embed
-    const embedItem = new MessageEmbed();
-    embedItem.setColor('#0099ff').setTitle(title);
+    let embedItem: MessageEmbed;
+
+    if (typeof arg === 'object') {
```

Comment:
i dont like this

Reply:
the idea in general or the typeof command?

Reply:
typeof, make an options object as optional not a multi type arg